src/assets/js/themes.js

```diff
@@ -1,49 +1,48 @@
 /* theme toggle buttons */
-(function() {
-    var enableToggle = function(btn) {
+(function () {
+    var enableToggle = function (btn) {
         btn.setAttribute("aria-pressed", "true");
     }
-
-    var disableToggle = function(btn) {
+    var disableToggle = function (btn) {
         btn.setAttribute("aria-pressed", "false");
     }
+    var setTheme = function (theme) {
+        document.documentElement.setAttribute('data-theme', theme);
+        window.localStorage.setItem("theme", theme);
+    }
 
+    var theme = window.localStorage.getItem("theme");
+    if (!theme || (theme !== "light" && theme !== "dark")) {
```

Comment:
```suggestion
    if (!theme) {
```

When the theme value is falsy we are not going to have it as `light` and `dark`?  I think we don't need those conditions?

Reply:
yes if the theme value is falsy it won't be either light or dark. Suppose the theme value is truthy then we are making sure that it must be either light or dark. Just an additional check only. In case the local store value accidentally changed to something else other than light or dark.